mondrian/src/main/java/mondrian/olap/Mondrian.xml

```diff
@@ -966,7 +966,7 @@ Revision is $Id$
         <Attribute name="datatype" required="false">
             <Doc>
                 The datatype of this measure:
-                String, Numeric, Integer, Boolean, Date, Time or Timestamp.
+                String, Numeric, Integer.
 
```

Comment:
Well, there's definitely a bug since we're throwing an exception when using a datatype that's specified as allowed.

Digging in a bit more- the only reason my test was passing is that I used the "type" attribute name instead of "datatype", so the boolean field was being treated as an integer.

I still see value in explicitly supporting those datatypes, but I guess that becomes more of an enhancement.  I'll go ahead and merge this one.